server/routes/profileview.py

```diff
@@ -24,5 +24,5 @@ def get_profile_view(username):
             "registration_datetime": user_profile.registration_datetime,
             "total_tickets": total_tickets
         }
-    except:
+    except NoResultFound:
         return {"error": 500}
```

Comment:
You could return the error code like this:
`return {"error": "Result not found"}, 404`

Reply:
I have applied the change accordingly.